crates/astria-cli/src/cli/rollup.rs

```diff
@@ -0,0 +1,291 @@
+    #[clap(long, env = "ROLLUP_FAUCET_PRIVATE_KEY")]
+    pub(crate) faucet_private_key: String,
+    /// Sequencer private key
+    #[clap(long, env = "ROLLUP_SEQUENCER_PRIVATE_KEY")]
+    pub(crate) sequencer_private_key: String,
+}
+
+#[derive(Args, Debug)]
+pub struct DeploymentDeleteArgs {
+    /// The filepath of the target deployment's config
+    #[clap(long = "config")]
+    pub(crate) config_path: String,
+}
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use assert_cmd;
+
+    use super::*;
+    use crate::test_utils::with_temp_directory;
+
+    #[test]
+    fn test_genesis_account_arg_from_str_with_balance() {
+        let input = "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30:1000";
+        let expected = GenesisAccountArg {
+            address: "aC21B97d35Bf75A7dAb16f35b111a50e78A72F30".to_string(),
+            balance: 1000,
+        };
+        let result: GenesisAccountArg = input.parse().unwrap();
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_genesis_account_arg_from_str_without_balance() {
+        let input = "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30";
+        let expected = GenesisAccountArg {
+            address: "aC21B97d35Bf75A7dAb16f35b111a50e78A72F30".to_string(),
+            balance: 0,
+        };
+        let result: GenesisAccountArg = input.parse().unwrap();
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_genesis_account_arg_from_str_invalid_balance() {
+        let input = "0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30:invalid_balance";
+        let result: Result<GenesisAccountArg, _> = input.parse();
+        assert!(result.is_err());
+    }
+
+    #[test]
+    fn test_genesis_account_arg_from_str_invalid_address() {
+        let input = "0x1234abcd:1000";
+        let result: Result<GenesisAccountArg, _> = input.parse();
+        assert!(result.is_err());
+    }
+
+    #[test]
+    fn test_genesis_account_arg_from_str_no_0x_prefix() {
+        let input = "aC21B97d35Bf75A7dAb16f35b111a50e78A72F30:1000";
+        let expected = GenesisAccountArg {
+            address: "aC21B97d35Bf75A7dAb16f35b111a50e78A72F30".to_string(),
+            balance: 1000,
+        };
+        let result: GenesisAccountArg = input.parse().unwrap();
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_envars_are_parsed_for_config_create() {
+        with_temp_directory(|_dir| {
+            let mut cmd = assert_cmd::Command::cargo_bin(env!("CARGO_PKG_NAME")).unwrap();
+
+            cmd.arg("rollup")
+                .arg("config")
+                .arg("create")
+                .env("ROLLUP_USE_TTY", "true")
+                .env("ROLLUP_LOG_LEVEL", "debug")
+                .env("ROLLUP_NAME", "testtest")
+                .env("ROLLUP_CHAIN_ID", "test_chain_id")
+                .env("ROLLUP_NETWORK_ID", "53")
+                .env("ROLLUP_SKIP_EMPTY_BLOCKS", "true")
+                .env("ROLLUP_GENESIS_ACCOUNTS", "0x1234abcd:1000")
```

Comment:
Is this not an invalid address?

Reply:
Oh good catch! I missed this because these two envar tests have been failing in ci for an unrelated reason I think.